docs/misc/sprite-animator.mdx

```diff
@@ -15,7 +15,71 @@ sourcecode: src/core/SpriteAnimator.tsx
 
 ```tsx
 
-/** Common properties that can be safely applied to both sprite and instance components */
+/**
+ * Source size information for a sprite frame
+ */
+interface SourceSize {
+  /** Width of the source frame */
+  w: number
+  /** Height of the source frame */
+  h: number
+}
+
+/**
+ * Data structure for a single frame in the sprite sheet
+ */
+interface FrameData {
```

Comment:
I don't think users have to know about all those type, only the Props one

Reply:
cleaned them up and reduced the comments to the bare minimum on the documentation